src/osd/osd_types.h

```diff
@@ -50,6 +50,7 @@
 #include "common/hobject.h"
 #include "common/snap_types.h"
 #include "common/strtol.h" // for ritoa()
+#include "common/ceph_mutex.h" //To use make_mutex() for crimson
```

Comment:
nit: I think that the comment is confusing and should be removed. make_mutex is needed for both.

Classic was ok without it because of it being included indirectly (implicitly).